c/build_all/windowsce/build.cmd

```diff
@@ -0,0 +1,222 @@
+@REM Copyright (c) Microsoft. All rights reserved.
+@REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+@setlocal EnableExtensions EnableDelayedExpansion
+rem echo off
+
+set current-path=%~dp0
+rem // remove trailing slash
+set current-path=%current-path:~0,-1%
+
+set build-root=%current-path%\..\..
+rem // resolve to fully qualified path
+for %%i in ("%build-root%") do set build-root=%%~fi
+
+rem -----------------------------------------------------------------------------
+rem -- check prerequisites
+rem -----------------------------------------------------------------------------
+
+rem // check that SDK is installed
+
+rem -----------------------------------------------------------------------------
+rem -- parse script arguments
+rem -----------------------------------------------------------------------------
+
+rem // default build options
+set build-clean=0
+set build-config=Debug
+set build-platform=WindowsCE
+set CMAKE_run_e2e_tests=OFF
+set CMAKE_run_longhaul_tests=OFF
+
+:args-loop
+if "%1" equ "" goto args-done
+if "%1" equ "-c" goto arg-build-clean
+if "%1" equ "--clean" goto arg-build-clean
+if "%1" equ "--config" goto arg-build-config
+if "%1" equ "--platform" goto arg-build-platform
+if "%1" equ "--run-e2e-tests" goto arg-run-e2e-tests
+if "%1" equ "--run-longhaul-tests" goto arg-longhaul-tests
+call :usage && exit /b 1
+
+:arg-build-clean
+set build-clean=1
+goto args-continue
+
+:arg-build-config
+shift
+if "%1" equ "" call :usage && exit /b 1
+set build-config=%1
+goto args-continue
+
+:arg-build-platform
+shift
+if "%1" equ "" call :usage && exit /b 1
+set build-platform=%1
+goto args-continue
+
+:arg-run-e2e-tests
+set CMAKE_run_e2e_tests=ON
+goto args-continue
+
+:arg-longhaul-tests
+set CMAKE_run_longhaul_tests=ON
+goto args-continue
+
+:args-continue
+shift
+goto args-loop
+
+:args-done
+
+rem -----------------------------------------------------------------------------
+rem -- build with CMAKE and run tests
+rem -----------------------------------------------------------------------------
+
+rmdir /s/q "%USERPROFILE%\cmake_ce8"
+rem no error checking
+
+mkdir "%USERPROFILE%\cmake_ce8"
+rem no error checking
+
+pushd "%USERPROFILE%\cmake_ce8"
+
+set SDKNAME=TORADEX_CE800
```

Comment:
Is there anything specific to TORADEX needed here? I think all of this should be generic enough to build for anybody else wanting to use CE2013, right?